src/components/Dashboard/ModuleLink.tsx

```diff
@@ -203,7 +203,7 @@ const ModuleLink = ({ link }: { link: ModuleLinkInfo }): JSX.Element => {
     py: link.pyOc,
     showAll: maxLangOc,
   };
-  const isMissingLang = langToOc[userLang] ?? 0 < maxLangOc;
+  const isMissingLang = langToOc[userLang]! < maxLangOc;
```

Comment:
@danielzsh hm I haven't seen this syntax before, can you explain what changed?

Reply:
I'm actually not quite sure why this fix works, as the only case where `langToOc[userLang] ?? 0` differs from `langToOc[userLang]!` is when `langToOc[userLang]` is null; however, `null` and `0` should compare the same? (as in `null < x` is the same as `0 < x`)

Reply:
can you print out exactly what's being compared before and after the change?

Reply:
Oh yikes, it was an operator precendence issue...thanks for helping me catch that 😅 (btw would you mind re-approving)